src/Microsoft.DotNet.Tools.Pack/InputOutputPathsCalculator.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.ProjectModel;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Tools.Pack
+{
+    public class InputOutputPathsCalculator
```

Comment:
ArtifactPathsCalculator? InputOutputPathsCalculator seems funny to me. I think this implies more of a relationship than exists between them. This just tells us where to find input artifacts and to put output artifacts.